com.unity.perception/Editor/GroundTruth/AddToConfigWindow.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.Perception.GroundTruth;
+using UnityEngine.UIElements;
+using Object = UnityEngine.Object;
+using Random = UnityEngine.Random;
+
+namespace UnityEditor.Perception.GroundTruth
+{
+    class AddToConfigWindow : EditorWindow
+    {
+        private VisualElement m_Root;
```

Comment:
Have you setup the Unity coding syntax configuration in your IDE? I think the private keyword is implicit when using the standing Unity C# syntax.